app/components/c-product-detail.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div :class="b()">
+
+    <div :class="b('inner')">
+
+      <section :class="b('product')"> <pre>{{ product }}</pre></section>
+
+      <div :class="b('content')">
+
+        <div :class="b('main')">
+          <section :class="b('details')"> details</section>
+          <section :class="b('related')"> related</section>
+          <section :class="b('accessories')"> accessories</section>
+        </div>
+
+        <div :class="b('sidebar')">
+          <aside :class="b('recommendation')"> recommendation</aside>
+        </div>
+
+      </div>
+
+    </div>
+
+  </div>
+
+</template>
+
+<script>
+  import { mapGetters } from 'vuex';
+
+  export default {
+    name: 'c-product-detail',
+
+    // mixins: [],
+
+    // props: {},
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      ...mapGetters({
+        product: 'product/getProduct'
+      })
+
+    }
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    // methods: {},
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .c-product-detail {
```

Comment:
- [ ] Please also add fat, horizontal and vertical borders (they are no gaps). https://projects.invisionapp.com/d/main#/console/13428219/283490971/preview

Reply:
- [ ] Christoph also told me, that the spacing on bottom is intentional. So please add a `padding-bottom` to `main` and `sidebar` according to the `main` column in the design so the last line in `main` and the yellow line in the `sidebar` will never touch the edge of the content: https://projects.invisionapp.com/d/main#/console/13428219/283490971/preview